developer/js/build.sh

```diff
@@ -7,7 +7,9 @@
 set -eu
 
 # Include some helper functions from resources
-. ../../resources/shellHelperFunctions.sh
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+KEYMAN_ROOT="$(dirname "$THIS_SCRIPT")/../.."
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
```

Comment:
Please note the standard build script header format; @mcdurdin's been a big stickler on this point.

Refer to https://github.com/keymanapp/keyman/blob/master/web/source/build.sh#L6-L11 for an example that _also_ includes `shellHelperFunctions.sh`.  (`shellHelperFunctions.sh` may be left out of it, but not `build_utils.sh`.)

Reply:
Hey if we really have to include a messy hoopy piece of boilerplate, I really want it to be as close to identical as possible in all references. Because then a grep can update it in the future 😁 

(FWIW the whole greadlink/readlink mess is yet more cross platform stuff -- this time macOS uses an old version of readlink)